contracts/interfaces/IDelegation.sol

```diff
@@ -0,0 +1,44 @@
+pragma solidity 0.5.16;
+
+import "./IContractRegistry.sol";
+
+/// @title Elections contract interface
+interface IDelegations {
+    // Delegation state change events
+	event StakeChanged(address addr, uint256 stake);
```

Comment:
So this is emitted from both delegations and staking contracts?

Reply:
Removed